whylog/log_reader/exceptions.py

```diff
@@ -13,3 +13,11 @@ def __str__(self):
         return 'Config did not match any log_type to the specified front_input: %s; cannot investigate' % (
             self._front_input
         )
+
+
+class ReadingError(LogReaderError):
```

Comment:
this should be two classes, one for empty file and one for offset bigger than file size